CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+## 2.0.0
+
+### Added
+
+- Support from HTTPlug 2.0 and PSR-18
+
+### Changed
+
+- `Client` and `Promise` are both final. 
+
+## Removed
+
+- Support for PHP <7.1
```

Comment:
This should go into 1.2 version IMHO.

Reply:
How come?

Reply:
Well, not sure. Looking at the diff between the latest release and the current master there are quite a few changes. But as I took a closer look, nothing special, so we might not have to release 1.2 after all. I had the impression from the number of commits that we should probably release a last 1.x version before releasing 2.0